### PR TITLE
added circular annulus subset to cubviz toolbar

### PR DESCRIPTION
**Pull request by cshanahan1**
This PR adds a circular annulus subset option to the toolbar.

### CODE REVIEW

**Comment by javerbukh** [comment]
Works for me! If you can fix the one code style check we can see if it passes CI as well. I can approve after that.

**Comment by camipacifici** [comment]
Tested in standalone. Works great. Thank you!

File: jdaviz/configs/cubeviz/plugins/viewers.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class CubevizImageView(JdavizViewerMixin, BqplotImageView):
                     ['jdaviz:homezoom', 'jdaviz:prevzoom'],
                     ['jdaviz:pixelboxzoommatch', 'jdaviz:boxzoom',
                      'jdaviz:pixelpanzoommatch', 'jdaviz:panzoom'],
-                    ['bqplot:truecircle', 'bqplot:rectangle', 'bqplot:ellipse'],
+                    ['bqplot:truecircle', 'bqplot:rectangle', 'bqplot:ellipse', 'bqplot:circannulus'],
```
Comment (kecnry) [inline jdaviz/configs/cubeviz/plugins/viewers.py:27]:
```suggestion
                    ['bqplot:truecircle', 'bqplot:rectangle',
                     'bqplot:ellipse', 'bqplot:circannulus'],
```

**Review by kecnry — APPROVED** [review]

**Comment by cshanahan1** [comment]
thanks! ill merge once CI passes

**Pull request merged**